apps/prairielearn/src/lib/externalGrader.ts

```diff
@@ -232,6 +242,15 @@ export async function processGradingResult(content: any): Promise<void> {
       content.grading.score,
       null, // `v2_score`: gross legacy, this can safely be null
     ]);
+
+    if (updateUsages) {
+      // This has to come after `grading_jobs_update_after_grading` above
+      // because it uses the `grading_finished_value` updated there.
```

Comment:
I think you meant this?

```suggestion
      // because it uses the `grading_finished_at` value updated there.
```

Reply:
Yes, thanks!